package/Rakefile

```diff
@@ -176,6 +178,117 @@ class MilterManagerTask < PackagesGroongaOrgPackageTask
     end
     task :source => source_tasks.collect {|task| "source:#{task}"}
   end
+
+  def apply_obs_pkg_metadata(project_name, package_name)
+    Dir.chdir(project_name) do
+      require 'tempfile'
+      Tempfile.create("meta.xml") do |f|
+        f.puts(<<-EOS
+<package name="#{package_name}" project="#{project_name}">
+  <title>Milter manager #{detect_version}</title>
+  <description/>
+</package>
+EOS
+              )
+        f.flush
+        sh("osc", "meta", "pkg", package_name, "--file", f.path)
+      end
+      sh("osc", "update")
+    end
+  end
+
+  def obs_project_name
+    ENV["OBS_PROJECT"] ? "isv:milter-manager" : "home:#{ENV['USER']}"
+  end
+
+  def obs_package_dir(package_name)
+    File.join(obs_project_name, package_name)
+  end
+
+  def copy_obs_source_files(repositories_dir, package_name, pattern)
+    files = []
+    repositories_dir.glob(pattern) do |path|
+      relative_path = path.relative_path_from(repositories_dir)
+      # e.g.: "debian/pool/bullseye/main/m/milter-manager/milter-manager*.dsc"
+      # e.g.: "centos/7/SRPMS/milter-manager*.src.rpm"
+      files << path.basename.to_s
+      cp(path.to_s, obs_package_dir(package_name))
+    end
+    files
+  end
+
+  def define_apt_obsbuild_tasks
+    obsbuild_tasks = []
+    namespace :apt do
+      namespace :obsbuild do
+        obsbuild_tasks << :source
+        obsbuild_tasks << :package
+
+        desc "Build source package"
+        task :source => [] do
+          # Enable building source package
+          ENV["SOURCE_BUILD"] = "yes"
+          apt_build
+        end
```

Comment:
Could you use built artifacts instead?
You can find built artifacts here: https://github.com/milter-manager/milter-manager/actions/runs/4060065096#artifacts

I don't recall but `rake apt:download` or something download them automatically.